### ISSUE TITLE
Breaking error Foundry 0.8.6 (after switching between scenes)

### ISSUE THREAD

**Issue by amcvfx** [itamarcu/AlternativeRotation#8]
I am getting an error that doesn't always occur but stops Alternative Rotation working when it does crop up:
I'm on Foundry 0.8.6

![image](https://user-images.githubusercontent.com/35099081/121231809-c61f9c80-c888-11eb-966b-c77ff3e8cefd.png)

**Comment by itamarcu** [comment]
@amcvfx - just to be sure, are you using v1.2.2 of the module?

**Comment by vlaminck** [comment]
I'm using 1.2.2 and have this error as well. On screen load, I see your `Alternative Rotation | added PIXI graphics to canvas controls` log message. 

When I try to shift+drag to rotate on the activated scene, it works. If I navigate to a different scene and try to rotate, I get these errors


```
Uncaught TypeError: Cannot read property 'clear' of null
    at r.clear (graphics.min.js:8)
    at drawDirectionalArrow (alternative-rotation.js:73)
    at MouseInteractionManager._handleDragMove_Override (alternative-rotation.js:230)
    at z.🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:558)
    at MouseInteractionManager.🎁MouseInteractionManager.prototype._handleDragMove#0 (libWrapper-wrapper.js:145)
    at MouseInteractionManager._handleMouseMove (foundry.js:35694)
    at e.a.emit (index.js:202)
    at r.dispatchEvent (interaction.min.js:8)
    at r.processPointerMove (interaction.min.js:8)
    at t.recursiveFindHit (interaction.min.js:8)
```

```
Uncaught TypeError: Cannot read property 'clear' of null
    at r.clear (graphics.min.js:8)
    at MouseInteractionManager._handleDragCancel_Override (alternative-rotation.js:287)
    at z.🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:558)
    at MouseInteractionManager.🎁MouseInteractionManager.prototype._handleDragCancel#0 (libWrapper-wrapper.js:145)
    at MouseInteractionManager._handleMouseUp (foundry.js:35743)
    at z.🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:449)
    at MouseInteractionManager._handleMouseUp_Override (alternative-rotation.js:269)
    at z.🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:558)
    at MouseInteractionManager.🎁MouseInteractionManager.prototype._handleMouseUp#0 (libWrapper-wrapper.js:145)
    at Token5e.a.emit (index.js:181)
```

If I navigate back to the active scene it still gives me those errors. So the rotation arrow is only drawn after the page initially loads. Navigating to a different scene breaks it. When I stop dragging, the token still rotates as expected, but it doesn't rotate as I drag, and the arrow is not drawn. I can reproduce this consistently so let me know if there's anything I can do to help debug it.

**Comment by itamarcu** [comment]
Thanks, @vlaminck, for the details!

I was able to reproduce it by switching between scenes and now I've pinpointed the problem and found a solution (implementing now).

**Comment by itamarcu** [comment]
Fixed in v1.2.3!

(please reopen this issue if I'm wrong)

**Comment by vlaminck** [comment]
Confirmed that 1.2.3 fixed it for me. Thanks for the quick fix!
